src/Integrations/Integrations/Mysqli/MysqliIntegration.php

```diff
@@ -183,15 +183,20 @@ function (SpanData $span) use ($integration) {
      * @param SpanData $span
      * @param string $name
      * @param string $resource
+     * @param $result
      */
-    public function setDefaultAttributes(SpanData $span, $name, $resource)
+    public function setDefaultAttributes(SpanData $span, $name, $resource, $result = null)
     {
         $span->name = $name;
         $span->resource = $resource;
         $span->type = Type::SQL;
         $span->service = 'mysqli';
         $span->meta[Tag::SPAN_KIND] = 'client';
         $span->meta[Tag::COMPONENT] = MysqliIntegration::NAME;
+
+        if ($result && property_exists($result, 'num_rows')) {
```

Comment:
Queries without resultset return true, and will throw a type error here:
```suggestion
        if (is_object($result) && property_exists($result, 'num_rows')) {
```

Reply:
changed this